### DISCUSSION TITLE
Topluluğa Katılım Kuralları

### DISCUSSION THREAD

**Discussion by muhtalipdede** [thecoderverse/community#7] [Ideas]
Merhabalar,

Açık kaynak projelerimize katkı sağlamalak isteyenler için gerekli kuralları belirlemek istiyoruz. Konu hakkında fikirlerinizi bu discussion altında tartışabiliriz.

**Comment by AimTune** [comment]
Merhaba,

[Flutter Rehberi](https://github.com/flutterturkey/flutter-rehberi) projesinde şöyle bir template kuralı yapmışlardı contribution işlemleri için [buradaki](https://github.com/flutterturkey/flutter-rehberi/pull/19) PR'dan inceleyebiliriz.
